action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
   args:
     description: 'Arguments to pass to Dagger'
     required: false
+  cmds:
+    description: 'Commands to run on Dagger'
+    required: false
+    deprecationMessage: 'Use cmds input instead'
```

Comment:
`deprecationMessage` on `args`, not `cmds`.

Reply:
oops, my bad, great catch!